decidim-proposals/app/views/decidim/proposals/proposals/_endorsements_count.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= link_to "#list-of-endorsements", id: "proposal-#{proposal.id}-endorsements-count", class: "button small compact light button--sc button--shadow #{fully_endorsed ? 'success' : 'secondary'}" do %>
+<%= link_to "#{proposal.proposal_endorsements_count.positive? ? "#list-of-endorsements" : "javascript:;"}", id: "proposal-#{proposal.id}-endorsements-count", class: "button small compact light button--sc button--shadow #{fully_endorsed ? 'success' : 'secondary'}" do %>
```

Comment:
Maybe we should use `link_to_if` here or not render the link at all?

Reply:
I can use a variable if you don't like this inliner:
```
<% link_href= proposal.proposal_endorsements_count.positive? ? "#list-of-endorsements" : "javascript:;" %>
 +<%= link_to link_href, id: "proposal-#{proposal.id}-endorsements-count", class: "button small compact light button--sc button--shadow #{fully_endorsed ? 'success' : 'secondary'}" do %> 
```

Reply:
The problem is not the inline ternary operator, but the link to `javascript:;`, which looks weird to me

Reply:
Indeed, but it is solution to the problem that avoids having to hack the CSSs.

Reply:
@tramuntanal You don't need to hack the CSS here. You can apply those classes to a `div` and it will look exactly the same 😄